class.jetpack-client.php

```diff
@@ -299,11 +299,7 @@ static function wpcom_json_api_request_as_blog( $path, $version = self::WPCOM_JS
 		$_path = preg_replace( '/^\//', '', $path );
 
 		// Use GET by default whereas `remote_request` uses POST
-		if ( isset( $filtered_args['method'] ) && strtoupper( $filtered_args['method'] === 'POST' ) ) {
```

Comment:
Oh! Looks like I'm also fixing this little bug: `strtoupper( $filtered_args['method'] === 'POST' )`. That's probably supposed to read `strtoupper( $filtered_args['method'] ) === 'POST' )`.